app/models/page.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Page < ApplicationRecord
+  include Validators
+  has_many_attached :documents, dependent: :destroy
```

Comment:
I thought we decided to do this as `has_one`? We'd have a page object for each org chart, rather than a page with many org charts.

Reply:
There may be instances of a page that would need more than one file attachment.

Reply:
Orgcharts come to mind, where we can list all the charts on one page, like an index, and then link to them.